File: 프로그래머스/0/120896. 한 번만 등장한 문자/한 번만 등장한 문자.py
def solution(s):
    freq = {}
    arr = []
    answer = ""
    
    for ch in s:
        freq[ch] = freq.get(ch, 0) + 1
    
    print(freq)
        
    for key, value in freq.items():
        if value == 1:
            arr.append(key)
    
    arr.sort()
    
    for i in arr:
        answer += i
        
    print(answer)
    
    return answer

File: 프로그래머스/0/120896. 한 번만 등장한 문자/한 번만 등장한 문자.swift
import Foundation

func solution(_ s:String) -> String {
    var dic:[Character:Int] = [:]
    var answer: String = ""
    
    for ch in s {
        dic[ch, default:0] += 1
    }
    
    for (key, value) in dic {
        if value == 1 {
            answer.append(key)
        }
    }
    var arr = Array(answer)
    arr.sort()
    
    answer.removeAll()
    answer = String(arr)
    
    print(answer)
    
    return answer
}